src/pages/VenuePage/VenuePage.tsx

```diff
@@ -207,19 +202,10 @@ const VenuePage: React.FC = () => {
     );
   }, [profile, user, venue]);
 
-  const venueIdFromParams = getQueryParameters(window.location.search)
-    ?.venueId as string;
-
+  // @debt Remove this once we replace currentVenue with currentVenueNG our firebase
   useConnectCurrentVenue();
   useConnectCurrentEvent();
   useConnectUserPurchaseHistory();
-  useEffect(() => {
```

Comment:
we have currentVenueConnect a few lines above

Reply:
fixes https://github.com/sparkletown/internal-sparkle-issues/issues/122